src/containers/MainForm/EventOptions.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import {
+  Select,
+} from 'antd';
+const { Option } = Select;
+
+export const renderEventOptions = (mode, state) => {
+  let lawmakerType;
+  if (mode === 'moc' && state === null) {
+    lawmakerType = 'fedRep';
+  } else if (mode === 'moc' && state !== null) {
+    lawmakerType = 'stateRep';
+  } else if (mode === 'candidate' && state === null) {
+    lawmakerType = 'fedCandidate';
+  } else if (mode === 'candidate' && state !== null) {
+    lawmakerType = 'stateCandidate';
+  }
+  switch (lawmakerType) {
```

Comment:
it would be a lot cleaner to just make 5 arrays, and map over them to make the option tags. The array could be returned by a selector.

Reply:
ok, i've moved these to a selector that returns the appropriate event options array - it will then render in index file - I deleted the EventOptions file as it is not needed anymore